src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/FeeStep.tsx

```diff
@@ -28,6 +29,7 @@ export const FeeStep = ({ isLastStep }: FeeStepProps) => {
   const translate = useTranslate()
   const [showFeeModal, setShowFeeModal] = useState(false)
 
+  const inputAmountUsd = useAppSelector(selectInputSellAmountUsd)
```

Comment:
For paranoia we should only even use values from the quote (see code comment under this line), so may need to create a selector `selectActiveQuoteSellAmountUsd` if it doesnt already exist

Reply:
Do we want to do this in a follow-up? This is the same logic as previously, only moved away to the parent component and drilled down, so could inherently produce risks instead of just moving things around

Reply:
sounds like a plan ser